internal/pkggraph/graph.go

```diff
@@ -5,25 +5,27 @@ import (
 	"sort"
 
 	"golang.org/x/tools/go/packages"
+	"golang.org/x/tools/go/vcs"
 
 	"github.com/loov/goda/internal/stat"
 )
 
 type Graph struct {
-	Packages map[string]*Node
-	Sorted   []*Node
+	Packages map[string]*GraphNode
+	Sorted   []*GraphNode
 	stat.Stat
 }
 
-func (g *Graph) AddNode(n *Node) {
+func (g *Graph) AddNode(n *GraphNode) {
 	g.Packages[n.ID] = n
 	n.Graph = g
 }
 
-type Node struct {
+type GraphNode struct {
```

Comment:
I would keep the name as `Node`, because that's the main computational unit.

Reply:
Sure. I avoided renaming this for a while but kept finding myself not remember what a "Node" was. However, if we move tree into its own package as you suggest below that would probably help.